### ISSUE TITLE
No licence?

### ISSUE THREAD

**Issue by jskeet** [hishamco/MarkdownViewEngine#1]
This looks like exactly the project I'm looking for in order to serve the [Noda Time](http://nodatime.org) user guides. However, in order to contribute to it - and even use it as a nuget package - I'd need there to be a suitable licence stating how it can be used. (I'd personally appreciate either Apache 2.0, MIT, or BSD, but obviously YMMV.)

**Comment by hishamco** [comment]
@jskeet I built this as passionate for ASP.NET Core, I wasn't thinking about the licenses at that time. I will change the license soon , everyone are welcome to contribute.
FYI I built another project on top of this it called [mDocs](http://github.com/hishamco/mDocs) which is very helpful if you looking for documentation.
It is pleasure for me to help you in [Noda Time](http://nodatime.org/) user guides (if you want)
